app/controllers/flowcommerce_spree/flow_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module FlowcommerceSpree
+  class FlowController < ActionController::Base
```

Comment:
Hm, I don't quite get why do we need a separate endpoint and controller? The WebhooksController could handle any events, we have to just implement service objects for processing the events.

Reply:
Oh I thought it would be easy for to avoid mixing the webhook logic with this endpoint where it just simply returns information related to the items. In other words. Leave the webhook controller to handle webhook related actions that performs some kind of change in our side. 

Since this new endpoint would be used as a way of validating if the order is still valid at some point during the checkout. I was thinking that it made more sense to have it a separate endpoint; However, I am open to reuse webhook controller if you still think it is better :)

Reply:
Hm, maybe I'm not understanding here. How exactly are purchases going to be validated by flow.io? Won't it use  events?

Reply:
Hm, I have studied Flow Console and their docs - seems that I was wrong and it is another workflow, indeed, not using events. So, your separate controller is great! :thumbsup:
Maybe give it a more descriptive name then - FlowController is too generic. Something like InventoryController or similar?